checker/http/http.go

```diff
@@ -241,6 +271,37 @@ func (h *HTTP) Check(ctx context.Context) (err error) {
 	return nil
 }
 
+// getTLSConfig prepares TLS config
+func (h *HTTP) getTLSConfig() (*tls.Config, error) {
+	cfg := tls.Config{
+		InsecureSkipVerify: h.insecureSkipTLSVerify,
+	}
+
```

Comment:
I think you should check `h.insecureSkipTLSVerify`. If it's true, return immediately and ignore the rest of the logic.